remotespark/kernels/kernelmagics.py

```diff
@@ -64,6 +66,7 @@ def __init__(self, shell, data=None, spark_events=None):
     @handle_expected_exceptions
     @_event
     def help(self, line, cell="", local_ns=None):
+        self._assure_cell_body_is_empty(KernelMagics.help.__name__, cell)
```

Comment:
There's no error for that (this has always been the case). I can add extra assertions for that.